src/router/routes.js

```diff
@@ -8,7 +8,7 @@ const routes = [
   },
   {
     path: '/:catchAll(.*)*',
-    component: () => import('pages/ErrorNotFound.vue'),
+    component: () => import('pages/ErrorNotFoundPage.vue'),
```

Comment:
no need, it's not a `q-page`

Reply:
I don't modify this because, we will have an next pull request to remove it.